src/commands/createVirtualMachine/VirtualMachineCreateStep.ts

```diff
@@ -5,14 +5,16 @@
 
 import { ComputeManagementClient, ComputeManagementModels } from 'azure-arm-compute';
 import { NetworkManagementModels } from 'azure-arm-network';
-import { Progress } from "vscode";
+import { MessageItem, Progress, window } from "vscode";
 import { AzureWizardExecuteStep, createAzureClient } from "vscode-azureextensionui";
 import { ext } from '../../extensionVariables';
 import { localize } from '../../localize';
 import { nonNullProp, nonNullValueAndProp } from '../../utils/nonNull';
 import { getSshKey } from "../../utils/sshUtils";
 import { IVirtualMachineWizardContext } from './IVirtualMachineWizardContext';
 
+const viewOutput: MessageItem = { title: 'View Output' };
```

Comment:
nit: why is this all the way up here? Suggest moving it closer to where it's used